apps/server-web/src/renderer/App.css

```
@@ -27,17 +27,51 @@ html.dark {
 @layer base {
   html {
     font-family: 'Poppins', sans-serif;
+    scrollbar-width: thin; /* Makes it thinner than native */
+    scrollbar-color: #888888 transparent; /* thumb color | track color */
```

Stylelint (17.3.0)

[error] Expected "#888888" to be "#888" (color-hex-length)

(color-hex-length)